src/App.js

```diff
@@ -16,7 +16,11 @@ class App extends Component {
     Chart.defaults.global.defaultFontColor = '#000';
     Chart.defaults.global.defaultFontSize = 16;
 
-    this.state = {historicalData: null, currency: "PHP"}
+    this.state = {
+      historicalData: null, 
+      currency: "PHP",
+      baseUrl: 'https://api.coindesk.com/v1/bpi/historical/close.json'
```

Comment:
Great! What do you think of setting `baseUrl` to `https://api.coindesk.com/`? This is so we can reuse this variable in case we need another endpoint from them.